dashboard/db/migrate/20150804174622_change_phase_to_int.rb

```diff
@@ -0,0 +1,28 @@
+class ChangePhaseToInt < ActiveRecord::Migration
+  def change
+    Workshops.find_each do |workshop|
+      phase = workshop.phase
+      new_phase = 0
+     if phase == 'Phase 1'
+       new_phase = 1
+     elsif phase == 'Phase 2'
+       new_phase = 2
+     elsif phase == 'Phase 2 Online'
+       new_phase = 3
+     elsif phase == 'Phase 3A'
+       new_phase = 4
+     elsif phase == 'Phase 3B'
+       new_phase = 5
+     elsif phase == 'Phase 3C'
+       new_phase = 6
+     elsif phase == 'Phase 3D'
+       new_phase = 7
+     elsif phase == 'Phase 4'
+       new_phase = 8
+     end
+      workshop.phase = new_phase
```

Comment:
This will replace any phase strings not in the list with 0, which could result in to data loss.  Please have the code fail with an assertion failure if the phase is not recognized.

Could you wrap this with a transaction so that either all of the updates will succeed or none of them will?